Makefile

```diff
@@ -50,6 +50,31 @@ package-apisix-rpm:
 		--url 'http://apisix.apache.org/'
 	rm -rf ${PWD}/build
 
+### build dashboard:
+.PHONY: build-dashboard-rpm
+build-dashboard-rpm:
```

Comment:
I think we can merge `build-apisix-rpm` and `build dashboard` to one, and merge `package-dashboard-rpm` and `package-apisix-rpm` to one.

We can do this after this PR was merged.

Reply:
so we can only add a new `Dockerfile` to build a new package, eg: `dockerfiles/Dockerfile.dashboard.rpm`.